client/app/components/composites/Menu/Menu.css

```diff
@@ -0,0 +1,85 @@
+.menu {
+  font-family: var(--topbarFontFamily); /* should come from topbar */
+  font-size: var(--Menu_fontSize);
+  flex-grow: 1;
+
+  &:focus,
+  & :focus {
+    outline: none;
+  }
+
+  &:hover .menuContent {
+    outline: none;
+    display: block;
+  }
+}
+
+.menuLabel {
+  display: inline-block;
+  font-size: var(--Menu_fontSize);
+  padding: 28px 24px;
+  white-space: nowrap;
+  cursor: pointer;
+
+  &:first-letter {
+    text-transform:capitalize;
+  }
+}
+
+.menuLabelIcon {
+  margin-right: var(--Menu_iconPadding);
+}
+.menuLabelDropdownIcon {
+  margin-left: 0.2857em;
+}
+
+.menuLabelDropdownIconOpen {
```

Comment:
But then again, I have "inside" component called "MenuLabelDropdown", so I'm not changing this atm.